src/poprox_storage/repositories/accounts.py

```diff
@@ -104,3 +114,30 @@ def fetch_subscribed_accounts(self):
         )
         account_ids = self._id_query(account_query)
         return self.fetch_accounts(account_ids)
+
+    def fetch_subscription_for_account(self, account_id):
```

Comment:
I think have type annotations is going to help us keep track of which methods do what in this early realm where we don't have consistent method names yet, so I'd probably add some on these methods

Reply:
Well. -- I was a bit unsure on this one. In theory account_id is a UUID, but I see it as str elsewhere in this file, and that doesn't seperate the different types of UUIDs. As for return value -- it's technically just a raw row which is currently only used as None/not None. 

I'm guessing if we really wanted to fix the return value that would be doable, but might involve actually changing the return value (should be doable without problematic side-effect) for the parameters, we should probably pick a "canonical" representation of an id.

Reply:
The canonical representation should be `UUID` with conversion to `str` as needed

Reply:
Hmm... I think fixing this will have to be another PR -- I found multiple `str` in there and decided to standardize against what was already there.